src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { peopleWithParentsInfo } from '../../helpers';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setIsLoading(true);
+        const response = await getPeople();
+
+        const peopleFromServer = peopleWithParentsInfo(response);
+
+        setPeople(peopleFromServer as Person[]);
+        setIsLoading(false);
```

Comment:
no biggie, but since it's not loading( setIsLoading(false) ) anymore both in the case of success and error, **finally** statement seems fitting, don't you think?